packages/codebytes/src/drawers.tsx

```diff
@@ -121,6 +121,7 @@ export const Drawers: React.FC<DrawersProps> = ({ leftChild, rightChild }) => {
           hideOnClose
           id="output-drawer"
           aria-labelledby="output-drawer-label"
+          role="region"
```

Comment:
this solved an axe issue that came up in the monolith when importing

Reply:
(also should we add a pr review requirement for the repo?)

Reply:
I was getting a repeated axe issue in portal app for this label, so I removed it from the component to see if it would resolve it in a new package version. And yes we should have a pr review requirement. Sorry, I was just trying to get it to pass! I know not good.

Reply:
totally fine, are you able to set up pr review requirement or should we ask jake for help?

Reply:
let me see if i can figure it out first, before i ask jake! i think i may have other questions for him about client-modules/portal app builds, so if we need permissions to set that up i'll ask him